.circleci/codegen_validation/normalize_yaml_fragment.py

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env python3
 
+import os
 import sys
+
+sys.path.append(os.path.join(sys.path[0], '..'))
+
```

Comment:
Can you please add a comment, why this one is needed?

Reply:
added